scripts/openshiftci-periodic-tests.sh

```diff
@@ -26,6 +26,8 @@ oc login -u developer -p password@123 --insecure-skip-tls-verify
 # Check login user name for debugging purpose
 oc whoami
 
+source ./scripts/openshiftci-config.sh
```

Comment:
This assumes that this script is always run using `./scripts/openshiftci-periodic-tests.sh` and will fail if we run it from elsewhere, no? Is it always the case?
How about determining the location of `openshiftci-periodic-tests.sh` and using it instead?

Reply:
Okay, with the different `make` invocations in these scripts, it should be okay this way. Never mind then.

Reply:
As there are some `make ...` just after, I assume the script is always started from the root directory